### PR TITLE
DOC: Remove to_filename from Getting started

### PR DESCRIPTION
**Pull request by fepegar**
As suggested by @matthew-brett in #945.

### CODE REVIEW

**Comment by matthew-brett** [comment]
Looks right to me!  @effigies - what do you think?

File: doc/source/gettingstarted.rst [hunk 1]
```diff
@@ -114,11 +114,7 @@ and all other values are set to resonable defaults.
 Saving this new image to a file is trivial.  We won't do it here, but it looks
 like::
 
-    img.to_filename(os.path.join('build','test4d.nii.gz'))
-
-or::
-
-    nib.save(img, os.path.join('build','test4d.nii.gz'))
+    nib.save(img, os.path.join('build', 'test4d.nii.gz'))
```
Comment (effigies) [inline doc/source/gettingstarted.rst:117]:
```suggestion
>>> nib.save(img, os.path.join('build', 'test4d.nii.gz'))  # doctest: +SKIP
```

**Comment by fepegar** [comment]
I like that. I didn't know what "We won't do it here" meant anyway.

**Comment by matthew-brett** [comment]
Thanks @fepegar for following up with this, and @effigies for the review.

**Pull request merged**